src/patchkernel/element.cpp

```diff
@@ -1224,7 +1224,7 @@ ConstProxyVector<long> Element::getVertexIds(ElementType type, const long *conne
 					continue;
 				}
 
-				std::size_t vertexSortIndex = uniqueVertexIds.size();
+				long vertexSortIndex = uniqueVertexIds.size();
```

Comment:
I would fix the declaration of uniqueVertexIds, it should become ```std::unordered_map<long, std::size_t> uniqueVertexIds;``` (after that change, there is a long that should become a std::size_t at line 1242).

Reply:
Sorry @marcocisternino , already fixed following the Andrea's suggestion. The fix should be coherent now. Please refer to the newest code